eng/scripts/Language-Settings.ps1

```diff
@@ -5,9 +5,9 @@ $packagePattern = "*.pom"
 $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/java-packages.csv"
 $BlobStorageUrl = "https://azuresdkdocs.blob.core.windows.net/%24web?restype=container&comp=list&prefix=java%2F&delimiter=%2F"
 
-function Get-java-PackageInfoFromRepo ($pkgPath, $serviceDirectory, $pkgName)
+function Get-java-PackageInfoFromRepo ($pkgDirectoryPath, $serviceDirectoryName)
```

Comment:
As we talked about the others lets leave the parameters along for stage 1 and just handle the case where pkgName is null.